alacritty.yml

```diff
@@ -262,6 +258,16 @@
   #  foreground: None
   #  background: None
 
+  # Information bar
```

Comment:
Still not a fan of `information bar`. Especially because for things like search it's used as an input not as an output for information.

Maybe just use the location of this "bar" as a name? So something like `footer bar`? The concepts of footers should be familiar to people already so they know where to look for it. Could also go with just `footer` but I think people would struggle with associating that directly to the bar at the bottom.